source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -1362,61 +1628,61 @@ GLSSharpNavierStokesSolver<dim>::integrate_particles()
           // domain. If this is the case, the particle won't be updated.
           IBParticle<dim> save_particle_state         = particles[p];
           bool            save_particle_state_is_used = false;
-          try
-            {
-              // Define the correction vector.
-              Tensor<1, 3> dv = residual_velocity * 1 / d_residual_dv;
+          // Define the correction vector.
+          Tensor<1, 3> dv = residual_velocity * 1 / d_residual_dv;
```

Comment:
Are you sure that d_residual_dv is always going to be non-zero?

Reply:
I added DBL_MIN to it.